netbout-web/src/main/java/com/netbout/email/EmAlias.java

```diff
@@ -145,20 +140,31 @@ public void email(final String email, final String link)
                     )
                     .with(
                         new EnHTML(
-                            Joiner.on('\n').join(
+                            StringUtils.join(
                                 new Markdown.Default().html(
-                                    "Please verify your new email:"
-                                ), "<br/>",
-                                String.format("<a href=%s>%s</a>", link, link)
+                                    StringUtils.join(
+                                        "Hi,<br/>Your notification e-mail ",
+                                        "address for [netbout](http://",
+                                        "www.netbout.com) has been changed. ",
+                                        "Please verify it by clicking ",
+                                        "[here](", link, ")."
+                                    )
+                                ),
+                                "If you did not change your e-mail address ",
+                                "you can ignore this message <br/>",
+                                "This is an automated e-mail, please ",
+                                "do not reply. If you have any questions, ",
+                                "submit an issue <a href=\"",
+                                "https://www.github.com/yegor256/netbout\"",
```

Comment:
@amihaiemil is it possible to move this URL into a `private static final` field instead of having it in the middle of a `String`?